README.md

```diff
@@ -290,6 +290,10 @@ Require a special comment, called a pragma, to be present in the file's first do
 
 Valid pragmas are `@prettier` and `@format`.
 
+#### `--insert-pragma`
```

Comment:
Could you add `<--` and `-->` around the docs for this feature? It's best to keep features hidden from the README until just before we release a version.

Reply:
Sound good, will do!